.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: node_js
 node_js:
-- '12'
+- '10'
```

Comment:
Since node 12 gave us issues with Angular and Gemini do we want this at node 10 or 11?

Reply:
I'm with version 10.13.0 - and everything is working fine ..so 10 sound good

Reply:
I've been using Angular 8 with v11 and it has worked also. No preference here for the node version.